light-blockchain/src/abstract_blockchain.rs

```diff
@@ -91,12 +99,36 @@ impl AbstractBlockchain for LightBlockchain {
 
     fn get_blocks(
         &self,
-        _start_block_hash: &Blake2bHash,
-        _count: u32,
-        _include_body: bool,
-        _direction: nimiq_blockchain::Direction,
+        start_block_hash: &Blake2bHash,
+        count: u32,
+        include_body: bool,
+        direction: nimiq_blockchain::Direction,
         _txn_option: Option<&Transaction>,
     ) -> Vec<Block> {
-        todo!() // IPTODO
+        // Light Blockchain can't return blocks with body
+        if include_body {
+            return vec![];
+        }
+        self.chain_store
+            .get_blocks(start_block_hash, count, direction)
+    }
+
+    /// Fetches a given number of macro blocks, starting at a specific block (by its hash).
+    /// It can fetch only election macro blocks if desired.
+    /// Returns None if given start_block_hash is not a macro block.
+    fn get_macro_blocks(
+        &self,
+        start_block_hash: &Blake2bHash,
+        count: u32,
+        include_body: bool,
+        direction: Direction,
+        election_blocks_only: bool,
+        _txn_option: Option<&Transaction>,
+    ) -> Option<Vec<Block>> {
+        if include_body {
+            return Some(vec![]);
```

Comment:
This early return breaks the comment that it returns `None` if start block is not a macro block, as this early return does not check for that.

Reply:
This is being addressed in #1167.